packages/xo-server-backup-reports/src/index.js

```diff
@@ -567,7 +571,6 @@ class BackupReportsXoPlugin {
       }
     }
 
-    const nTasks = log.tasks.length
     const nSuccesses = nTasks - nFailures - nSkipped - nInterrupted
```

Comment:
Subtracting failures, skips and interruptions to VM tasks seems incorrect.

Reply:
Why it's incorrect?

Reply:
What if 1 VM where successfully backed-up but two pools were missing?

Reply:
nSuccesses will be 1 and it's good for this PR because pools' tasks aren't supported yet.

Reply:
You are probably right, still, IMHO it's very complicated and prone to error, I think it would be much simpler to simply count successes instead, what do you think?

Reply:
But, as discussed, will not handle the particular case of the pool task, so, what do you suggest? What behavior you want?

Reply:
As long as `nTasks`, `nFailures`, `nSkipped` and `nInterrupted` are only updated for VM tasks, I'm ok with it.

Maybe we should simply reverse the count for `nTasks`: count `nSuccesses` instead and add all values to compute `nTasks`, IMHO it will be easier to understand and maintain, do you agree?